components/Ripple/tests/unit/MDCRippleViewUsesSuperviewShadowLayerAsMaskTests.m

```diff
@@ -108,4 +108,22 @@ - (void)testDoesNotInheritParentShadowPathWhenRippleStyleSetToUnbounded {
   XCTAssertTrue(CGPathEqualToPath(maskShapeLayer.path, parentView.layer.shadowPath));
 }
 
+- (void)
+    testDoesNotInheritParentShadowPathWhenRippleStyleSetToBoundedWhenUsesSuperviewShadowLayerAsMaskDisabled {
+  // Given
+  MDCRippleView *rippleView = [[MDCRippleView alloc] init];
+  UIView *parentView = [[UIView alloc] init];
+  [parentView addSubview:rippleView];
+  parentView.frame = CGRectMake(0, 0, 100, 50);
+  rippleView.frame = CGRectMake(10, 10, 30, 30);
+
+  // When
+  rippleView.usesSuperviewShadowLayerAsMask = NO;  // Disable the incorrect behavior.
```

Comment:
does order of operations matter?
What happens if you set this flag to NO after you set the style? My current read of the code: it wont remove the mask. Should it?

Reply:
Added some docs.